### DISCUSSION TITLE
Frigate road map

### DISCUSSION THREAD

**Discussion by izzygold** [blakeblackshear/frigate#13460] [General]
Hi,
Is there a place where I can see Frigate's roadmap for 0.15 (and further?)

like, for example, I know that there is a third party app for face detection with frigate, are there plans to have face detection integrated into frigate? (if not - I might like to clone the repo and try and add it myself, if there are plans, I won't do that...)

**Comment by NickM-27** [comment]
there is no public road map, we have an internal one but things change often so not sure if we want to make it public. Integration with face recognition directly is something that has been pinned and will be implemented at some point. 

If you want to make any contributions to frigate, especially larger ones, please create a discussion to detail the idea before putting up a PR

**Reply by izzygold** [reply]
Thanks for the answer
